moped-editor/src/views/projects/projectView/NoteInput.js

```diff
@@ -0,0 +1,260 @@
+import React, { useEffect } from "react";
+import {
+  Box,
+  Button,
+  Container,
+  Grid,
+  FormControl,
+  Radio,
+  RadioGroup,
+  FormControlLabel,
+} from "@mui/material";
+import makeStyles from "@mui/styles/makeStyles";
+import ProjectSaveButton from "../newProjectView/ProjectSaveButton";
+import ToolbarPlugin from "./ToolbarPlugin";
+
+import { AutoFocusPlugin } from '@lexical/react/LexicalAutoFocusPlugin';
+import { LexicalComposer } from '@lexical/react/LexicalComposer';
+import { RichTextPlugin } from '@lexical/react/LexicalRichTextPlugin';
+import { ContentEditable } from '@lexical/react/LexicalContentEditable';
+import { HistoryPlugin } from '@lexical/react/LexicalHistoryPlugin';
+import { LexicalErrorBoundary } from '@lexical/react/LexicalErrorBoundary';
+import { useLexicalComposerContext } from '@lexical/react/LexicalComposerContext';
+import { $generateHtmlFromNodes, $generateNodesFromDOM } from '@lexical/html';
+import { $getRoot } from 'lexical';
+import { $isRootTextContentEmpty } from '@lexical/text';
+import { LinkPlugin } from '@lexical/react/LexicalLinkPlugin';
+import { LinkNode } from '@lexical/link';
+import { ListPlugin } from '@lexical/react/LexicalListPlugin';
+import { ListNode, ListItemNode } from '@lexical/list';
+import PlaygroundEditorTheme from "./PlaygroundEditorTheme";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+    backgroundColor: theme.palette.background.paper,
+  },
+  cancelButton: {
+    margin: theme.spacing(1),
+    position: "relative",
+    color: theme.palette.secondary.dark,
+  },
+  contentEditable: {
+    width: "100%",
+    padding: "16px 16px",
+    border: "4px solid",
+    borderRadius: "8px",
```

Comment:
In other places where we want the nice MUI border styles, we have used [Paper](https://v5.mui.com/material-ui/react-paper/) component an elevation of 2. I think if you remove these border styles and wrap `<ContentEditable />` with Paper then it would match up nicely with the rest of the Notes UI.